crmd/utils.c

```diff
@@ -1008,6 +1008,55 @@ erase_status_tag(const char *uname, const char *tag, int options)
     }
 }
 
+static void
+crm_set_join_state(const char *uname, const char *start_state)
+{
+    if (safe_str_eq(start_state, "standby")) {
+        crm_notice("Starting node in %s state (%s)", start_state, uname);
+        update_attrd(uname, XML_CIB_ATTR_STANDBY, "on", NULL, FALSE);
+    } else if (safe_str_eq(start_state, "online")) {
+        crm_notice("Starting node in %s state (%s)", start_state, uname);
+        update_attrd(uname, XML_CIB_ATTR_STANDBY, "off", NULL, FALSE);
+    } else if (safe_str_eq(start_state, "default")) {
+        crm_notice("Starting node by default (%s)", uname);
+    } else {
+        crm_warn("Unrecognized start state '%s', using 'default' (%s)", start_state, uname);
+    }
```

Comment:
Minor request, I'd like to change the wording of the messages.

For the first two, maybe: "Forcing node %s to join in %s state per configured environment"

For "default", let's reduce the message to debug level since nothing is actually being done, and maybe say "Not forcing a starting state on node %s".